ownCloud/Viewer/WebViewDisplayViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ImageDisplayViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 30/08/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+import WebKit
+
+class WebViewDisplayViewController: DisplayViewController, DisplayExtension {
+	
+	static var displayExtensionIdentifier: String = "imageViewer"
```

Comment:
Analogous to `PDFViewerViewController`, a reverse-domain-name-style identifier here could be `org.owncloud.webview`.